sockaddr.go

```diff
@@ -33,7 +33,7 @@ func toSockaddrInet(ip net.IP, port int, zone string) (*C.struct_sockaddr, C.soc
 	if n := copy(rsa.Addr[:], ip); n != 16 {
 		panic(n)
 	}
-	return (*C.struct_sockaddr)(unsafe.Pointer(&rsa)), C.socklen_t(syscall.SizeofSockaddrInet6)
+	return (*C.struct_sockaddr)(unsafe.Pointer(&rsa)), C.socklen_t(unsafe.Sizeof(rsa))
```

Comment:
If this strategy works on both unix and windows, lets stick with it.

Reply:
yup, need revert, lets sort out the things about circle. and it will happen.

Reply:
https://circleci.com/gh/anacrolix/go-libutp

There's not currently a circle turd for this project as it's simple to test, but I'll add one eventually modelled on the one I have here: https://github.com/anacrolix/torrent/blob/master/.circleci/config.yml

Reply:
works for me. rm windows additions, rm .travis, add circleci. tbd.